news/2018-06-22-cwc-2018-recap-round-of-16.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "osu!catch World Cup 2018: Round of 16 Recap"
+date: 2018-06-22 12:00:00 +0000
+---
+
+Two weeks of frantic fruit-catching in the 2018 osu!catch World Cup completed, and several more still to look forward to - but who triumphed at the top of the tree, and who fell off the platter of progress in the Round of 16? Read on to find out more!
```

Comment:
`2018 osu!catch World Cup` -> `osu!catch World Cup 2018`

Reply:
Reversed, might used "2018 edition of..." in future